src/OrleansCodeGenerator/RoslynCodeGenerator.cs

```diff
@@ -484,6 +480,36 @@ private static void RecordType(Type type, Module module, Assembly targetAssembly
         }
 
         /// <summary>
+        /// Get types which have corresponding generated classes.
+        /// </summary>
+        /// <returns>Types which have corresponding generated classes marked.</returns>
+        private static HashSet<Type> GetTypesWithGeneratedSupportClasses()
+        {
+            // Get assemblies which contain generated code.
+            var all =
+                AppDomain.CurrentDomain.GetAssemblies()
+                    .Where(_ => _.GetCustomAttribute<GeneratedCodeAttribute>() != null)
+                    .SelectMany(_ => _.DefinedTypes);
```

Comment:
oh, right, I didn't realize it was using the singular version of the method... cool then.
Nah, if it was just for this change, then it's fine not to fix the `_` now. Just have it in the back of your mind for future work :)

Thanks, merging now